test/test_pmm.py

```diff
@@ -814,6 +829,18 @@ def test_config_spread_on_the_fly_multiple_orders(self):
         self.assertAlmostEqual(Decimal("96"), last_bid_order.price, 2)
         self.assertAlmostEqual(Decimal("104"), last_ask_order.price, 2)
 
+    def test_order_override(self):
+        strategy = self.one_level_strategy
+        self.clock.add_iterator(strategy)
+        self.clock.backtest_til(self.start_timestamp + self.clock_tick_size)
+
+        buys = strategy.active_buys
+        sells = strategy.active_sells
+        self.assertEqual(Decimal("99"), buys[0].price)
+        self.assertEqual(Decimal("1"), buys[0].quantity)
+        self.assertEqual(Decimal("101"), sells[0].price)
+        self.assertEqual(Decimal("1"), sells[0].quantity)
+
```

Comment:
There are 3 things you need to check for in order_override, spread, amount and order levels. Please test order levels here, e.g. `self.assertEqual(1, len(buys))`

Reply:
Yes, I added this to the unit test.